scripts/gear.sh

```diff
@@ -316,8 +316,8 @@ case "$COMMAND" in
         client_tests "$ROOT_DIR" "$@"; ;;
 
       fuzz)
-        header "Running fuzzer for system consistency check"
-        run_fuzzer "$ROOT_DIR" "$@"; ;;
+        header "Running fuzzer for runtime panick checks"
```

Comment:
```suggestion
        header "Running fuzzer for runtime panic checks"
```

Reply:
Wasn't this command used for economic checks?

Reply:
I'd separate commands

Reply:
economic checks are dead and will be migrated to `node-fuzzer` as invariants checks

Reply:
seems the crate can be deleted though?

Reply:
Well, I guess so. Shall we do it in this PR?

Reply:
Provide separate one once this merged pls, it's pretty simple